tembo-stacks/src/stacks/specs/analytics.yaml

```
@@ -0,0 +1,61 @@
+name: Parade Analytics
```

```suggestion
name: Analytics
```

Will be easier if it's one word no space.

---

Thought this was a display name for some reason; fixed it.